docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -189,6 +189,13 @@ NuGet then loads and applies settings as follows, depending on where it's invoke
 
 - **Invoked from disk_drive_2/Project2 or disk_drive_2/Project2/Source**: The user-level file (A) is loaded first followed by file (B) and file (D). Because `packageSources` is not cleared, both `nuget.org` and `https://MyPrivateRepo/DQ/nuget` are available as sources. Packages get expanded in disk_drive_2/tmp as specified in (B).
 
+## Additional user wide configuration
+
+Starting with 5.7, NuGet has added support for extra user wide configuration files. This capability was added in order to allow 3rd party vendors to add additional user configuration files without elevation.
```

Comment:
What do you think of these tweaks? I don't feel strongly about these, feel free to skip:

```suggestion
Starting with 5.7, NuGet has added support for additional user wide configuration files. This allows third-party vendors to add additional user configuration files without elevation.
```